posthog/hogql/database/schema/util/session_where_clause_extractor.py

```diff
@@ -0,0 +1,380 @@
+from dataclasses import dataclass
+from typing import Optional
+
+from posthog.hogql import ast
+from posthog.hogql.ast import CompareOperationOp, ArithmeticOperationOp
+from posthog.hogql.context import HogQLContext
+from posthog.hogql.database.models import DatabaseField
+from posthog.hogql.visitor import clone_expr, CloningVisitor, Visitor
+
+SESSION_BUFFER_DAYS = 3
```

Comment:
You should double check with the session folks, but there might have even been some 24h-ish limits on sessions. So moving this to `1` might be good enough, and faster.

Reply:
Last time I checked, there's no actual agreed upon limit, it's just 24 hours of inactivity. I'll leave this at 3 but I can change this later

Reply:
We now have a maximum session length of 24 hours, so I'll make a PR to change this number to 1.